metasyn/__main__.py

```diff
@@ -46,11 +50,72 @@ def main() -> None:
     elif subcommand == "schema":
         schema()
 
+    elif subcommand == "create-meta":
+        create_metadata()
+
     else:
         print(f"Invalid subcommand ({subcommand}). For help see metasyn --help")
         sys.exit(1)
 
 
+def _parse_config(config_fp):
+    config = ConfigParser()
+    config.read(config_fp)
+    spec = {}
+    for section in config.sections():
+        if section.startswith("var."):
+            new_dict = {}
+            for key, val in dict(config[section]).items():
+                try:
+                    new_dict[key] = config.getboolean(section, key)
+                except ValueError:
+                    pass
+                try:
+                    new_dict[key] = config.getfloat(section, key)
+                except ValueError:
+                    pass
+                try:
+                    new_dict[key] = config.getint(section, key)
+                except ValueError:
+                    pass
+                if key not in new_dict:
+                    new_dict[key] = val
+            spec[section[4:]] = new_dict
+    return spec
+
+
+def create_metadata():
+    """Program to create metadata from a dataframe."""
+    parser = argparse.ArgumentParser(
+        prog="metasyn create-meta",
+        description="Create a Generative Metadata Format file from a CSV file.",
```

Comment:
Might be nice to explain here that the input CSV has to be a tabular dataset? Not sure if the sentence would be too long then

Reply:
Isn't a CSV file always a tabular dataset though?

Reply:
I suppose they often are, you're right. That said, not everyone might be aware of that, in which case it is a lot more user-friendly to ask for an input tabular dataset, despite being redundant (IMO). It doesn't hurt, that's for sure. But if you like it more this way that is cool too.